src/components/schedule-page/table/schedule/schedule-parts/base-cell/base-cell.component.tsx

```diff
@@ -135,10 +136,14 @@ export function BaseCellComponent({
 
         {(!isPointerOn || (isPointerOn && isBlocked)) && (
           <p
+            style={{ position: "relative" }}
```

Comment:
trzyma trójkąt w miejscu

Reply:
nie jestem wielkim fanem styli inline. Dało by się to zrobić inaczej?

Reply:
wydawało mi się, że to za mało żeby robic osobną klasę, ale mogę.